Lib/Rabbit4000/RabbitSys/mem.lib

```diff
@@ -54,21 +54,21 @@ mem_debug
 __xmem char alloc_calc_chksum(void __far *p, int num) {
 #asm
 	ld hl, (sp+@sp+num)
+    ld bcde, 1
    ld b, l
    dec b ;; don't include checksum in calc
    xor a
    ld	c,a
-	ld px, (sp+@sp+p)
+   clr hl
 .loop:
-   ld a, (px)
+   ld a, (px+hl)

```

Comment:
I think there should be a comment here explaining the `+hl`.  I assume it's to maintain even alignment for faster execution, but it isn't obvious that it's equivalent to the previous code.